rundeckapp/webhooks/src/main/groovy/webhooks/responder/DefaultJsonWebhookResponder.groovy

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Rundeck, Inc. (http://rundeck.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package webhooks.responder
+
+import com.dtolabs.rundeck.plugins.webhook.WebhookResponder
+
+import javax.servlet.http.HttpServletResponse
+
+
+class DefaultJsonWebhookResponder implements WebhookResponder {
```

Comment:
maybe a default Json responder implementation would be useful in core, so plugin authors don't have to serialize json manually, just pass a Map/Collection to the default responder that uses jackson serialization